src/main/java/com/gw2auth/oauth2/server/service/user/Gw2AuthUserV2.java

```diff
@@ -24,16 +25,16 @@ public class Gw2AuthUserV2 implements OAuth2User, Principal, AuthenticatedPrinci
 
     private final UUID accountId;
     private final Pair<String, String> accountFederation;
-    private final String sessionId;
-    private final SessionMetadata sessionMetadata;
-    private final Instant sessionCreationTime;
-    private final byte[] encryptionKey;
+    private final @Nullable String sessionId;
+    private final @Nullable SessionMetadata sessionMetadata;
+    private final @Nullable Instant sessionCreationTime;
+    private final byte @Nullable [] encryptionKey;
```

Comment:
can the nullable be moved to the left hand side here?

Reply:
Paraphrasing from what I told you elsewhere:
- `@Nullable byte[]` would be `Array<Byte?>` in Kotlin,
- whereas `byte @Nullable []` would be `Array<Byte>?`